### ISSUE TITLE
IEV data anomaly: orphaned terms

### ISSUE THREAD

**Issue by ronaldtse** [glossarist/iev-data#38]
Orphaned terms (these “de” terms have no English/French counterparts)
```
[tc211-termbase] term (lang: deu, ID: 714-21-01) is missing a corresponding English term, probably needs updating.
[tc211-termbase] term (lang: deu, ID: 714-21-06) is missing a corresponding English term, probably needs updating.
[tc211-termbase] term (lang: deu, ID: 714-21-07) is missing a corresponding English term, probably needs updating.
[tc211-termbase] term (lang: deu, ID: 714-21-09) is missing a corresponding English term, probably needs updating.
[tc211-termbase] term (lang: deu, ID: 714-21-10) is missing a corresponding English term, probably needs updating.
[tc211-termbase] term (lang: deu, ID: 714-23-05) is missing a corresponding English term, probably needs updating.
[tc211-termbase] term (lang: deu, ID: 714-23-06) is missing a corresponding English term, probably needs updating.
[tc211-termbase] term (lang: deu, ID: 714-23-16) is missing a corresponding English term, probably needs updating.
```

<img width="444" alt="Screen Shot 2020-11-20 at 6 33 47 PM" src="https://user-images.githubusercontent.com/11865/99936781-ed0bfa80-2d9e-11eb-9a4a-5d3300280ff9.png">

195-04-13:

<img width="258" alt="Screen Shot 2020-11-20 at 6 35 17 PM" src="https://user-images.githubusercontent.com/11865/99936801-f72df900-2d9e-11eb-9145-a8db8daee533.png">

411-48-09:

<img width="393" alt="Screen Shot 2020-11-20 at 6 35 46 PM" src="https://user-images.githubusercontent.com/11865/99936806-fa28e980-2d9e-11eb-9cb9-b837f553c7db.png">

411-53-21, 411-53-22

<img width="350" alt="Screen Shot 2020-11-20 at 6 36 20 PM" src="https://user-images.githubusercontent.com/11865/99936817-ff863400-2d9e-11eb-8a90-aa9c6f8e2720.png">

723-01-14:

<img width="440" alt="Screen Shot 2020-11-23 at 3 19 24 PM" src="https://user-images.githubusercontent.com/11865/99936919-483ded00-2d9f-11eb-9008-831d7b559938.png">

807-01-07:

<img width="259" alt="Screen Shot 2020-11-23 at 3 20 34 PM" src="https://user-images.githubusercontent.com/11865/99936980-728faa80-2d9f-11eb-83da-42effa3f6893.png">

811-20-09:

<img width="302" alt="Screen Shot 2020-11-23 at 3 23 41 PM" src="https://user-images.githubusercontent.com/11865/99937129-e2059a00-2d9f-11eb-9e66-4f5a64373f51.png">

891-03-14, 891-03-15:

<img width="339" alt="Screen Shot 2020-11-23 at 3 24 20 PM" src="https://user-images.githubusercontent.com/11865/99937172-f9dd1e00-2d9f-11eb-9807-b93fd685d193.png">

**Comment by ronaldtse** [comment]
714-21 to 714-24:

> Sections 714-21 to 714-24 of IEC 60050-714:1992 were cancelled and replaced by IEC 60050-171:2018.
> This will be a human error.
> I have made these “de” entries obsolete in the IEV. Please feel free to delete them [...]

195-04-13:

> I can find no record of this entry ever having existed.
> I have made this “es” entry obsolete in the IEV. Please feel free to delete it [...]
 
411-48-09

> All other languages for this IEV ref (de, en, es, fr, it and ja) are obsolete in the IEV, and so I have made this entry obsolete in the IEV. Please feel free to delete it from the file I gave you.

411-53-21 and 411-53-22 are orphaned Polish and Portuguese terms.
> All other languages for this IEV ref (de, en, es, fr, it and ja) are obsolete in the IEV, and so I have made these entries obsolete in the IEV. Please feel free to delete them  [...]

604-0*-** only available in Slovenian

> IEC 60050-604 was Cancelled and replaced by IEC 60050-614:2016-02
> This will be a human error.
> I have made these “sl” entries obsolete in the IEV. Please feel free to delete them  [...]

723-01-14:
> I can find no other languages with this IEV ref.
> This IEV part dates back to 1997.
> IEC 60050-723 Amend. 1 © IEC:1999 says “Replace the existing term and definition 723-01-14 by the three terms and definitions 720-11-01, 720-11-02 and 720-11-03. See chapter 723-11. ” which is rather muddled since it talks about two different IEV numbers “720” and “723”.
> [...] since the de, it and ja exist in 723-11-01, 723-11-02 and 723-11-03, then these orphans can be made obsolete in the IEV. Please feel free to delete them [...]


807-01-07:

> The German text “kein eintrag in IEV” means “no entry in IEV” which is correct. This IEV ref does not exist in the published standard.
> I have made this “de” entry obsolete in the IEV. Please feel free to delete it [...]

811-20-09:

> All other languages for this IEV ref are obsolete in the IEV, and so I have made this entry obsolete in the IEV. Please feel free to delete it  [...]

891-03-14, 891-03-15:

> 891-03-14 and 891-03-15 do not exist in the published standard published in 1998.
> I will make these entries obsolete for now but will ask the JP NC whether they should be reinstated under another IEV ref.


Let's delete the relevant entries manually since they will be removed from the production database.

**Comment by ronaldtse** [comment]
These are verified fixed in #42.
